examples/python/ruff.toml

```diff
@@ -0,0 +1,228 @@
+    "SIM114",  # Combine if branches using logical or operator
+    "SIM115",  # Use a context manager for opening files
+    "SIM117",  # Use a single with statement with multiple contexts instead of nested with statements
+    "SIM118",  # Use key {operator} dict instead of key {operator} dict.keys()
+    "SIM300",  # Yoda condition detected
+    "SIM910",  # Use {expected} instead of {actual}
+    "SLF001",  # Private member accessed: {access}
+    "SLOT002", # Subclasses of {namedtuple_kind} should define __slots__
+    "T201",    # print found
+    "TC001",   # Move application import {} into a type-checking block
+    "TC003",   # Move standard library import {} into a type-checking block
+    "TD002",   # Missing author in TODO; try: # TODO(<author_name>): ... or # TODO @<author_name>: ...
+    "TD003",   # Missing issue link for this TODO
+    "TD004",   # Missing colon in TODO
+    "TD005",   # Missing issue description after TODO
+    "TRY002",  # Create your own exception
+    "TRY003",  # Avoid specifying long messages outside the exception class
+    "TRY004",  # Prefer TypeError exception for invalid type
+    "TRY203",  # Remove exception handler; error is immediately re-raised
+    "TRY300",  # Consider moving this statement to an else block
+    "TRY301",  # Abstract raise to an inner function
+    "TRY400",  # Use logging.exception instead of logging.error
+    "TRY401",  # Redundant exception object included in logging.exception call
+    "UP004",   # Class {name} inherits from object
+    "UP006",   # Use {to} instead of {from} for type annotation
+    "UP007",   # Use X | Y for type annotations
+    "UP008",   # Use super() instead of super(__class__, self)
+    "UP009",   # UTF-8 encoding declaration is unnecessary
+    "UP010",   # Unnecessary __future__ import {import} for target Python version
+    "UP015",   # Unnecessary mode argument
+    "UP018",   # Unnecessary {literal_type} call (rewrite as a literal)
+    "UP024",   # Replace aliased errors with OSError
+    "UP028",   # Replace yield over for loop with yield from
+    "UP030",   # Use implicit references for positional format fields
+    "UP031",   # Use format specifiers instead of percent format
+    "UP032",   # Use f-string instead of format call
+    "UP035",   # Import from {target} instead: {names}
+    "UP036",   # Version block is outdated for minimum Python version
+    "UP037",   # Remove quotes from type annotation
+    "UP045",   # Use X | None for type annotations
+    "YTT201",  # sys.version_info[0] == 3 referenced (python4), use >=
+    "YTT203",  # sys.version_info[1] compared to integer (python4), compare sys.version_info to tuple
+]
+
+exclude = [
+    "*_pb2.py",
+    "*_pb2_grpc.py"
+]
+
+# This is to suppress
+# https://docs.astral.sh/ruff/rules/non-pep585-annotation/
+# https://docs.astral.sh/ruff/rules/non-pep604-annotation-union/#non-pep604-annotation-union-up007
+pyupgrade.keep-runtime-typing = true
```

Comment:
instead, place it in `[lint.pyupgrade]` like in this example: https://docs.astral.sh/ruff/settings/#lint_pylint_max-statements